### ISSUE TITLE
New release ?

### ISSUE THREAD

**Issue by throwaway217** [akrennmair/newsbeuter#525]
The last release was 2 years ago. Are there any thoughts of snapping a release tag on a commit and let some new code hit the distros ?

**Comment by Minoru** [comment]
Oh, there *are* thoughts for sure, but:

* I don't want to create a tag without doing a release on newsbeuter.org because it'll create confusion in a number of ways; most importantly it'll mean we have a new release but search engines will still give users old docs.

* newsbeuter.org is managed by @akrennmair who haven't been responding to my emails for nine months now. (I even emailed his work address; no luck.) Noone else has access to the domain as far as I know.

* I really, really don't want to make a fork. It's quite a chunk of work, and doing it while the original author is still alive, well, and drinking beer seems premature.

* I got a job a few months ago and life's still a bit hectic. I don't have energy for upheavals right now. (Though it was already running low even before the job, because working on Newsbeuter without knowing if we'll release anything ever turned out to be quite demotivating.)

So there we are.

**Comment by throwaway217** [comment]
You could just mention all this in the release notes and just fork the documentation and put it up on a githup page.

**Comment by Minoru** [comment]
newsbeuter.org has been around for a decade and is linked from all over the Internet (and even from our own `--version`). GitHub page will take ages to overtake it; and by the time it does, I hope to finally prod Andreas into either transferring the domain or becoming more involved.

**Comment by throwaway217** [comment]
Does it really need to ? You could just employ this a temporary solution which may or may not become permanent. It's been 9 months. Why do we have so many cases of "what happens when you get hit by a bus" in the open source community when you can't push a new release forward because of this.

**Comment by Minoru** [comment]
> Does it really need to ?

Yes, otherwise the GitHub page will be useless (users won't see it).

> You could just employ this a temporary solution which may or may not become permanent.

I don't see a tag and docs on some unknown site as a solution. To me, a solution will mean that either newsbeuter.org is active again, or a fork occurs and new project does releases at its own home.

**Comment by throwaway217** [comment]
I think I understand your point of view. I have nothing more to add, thanks for your responses.

**Comment by fornwall** [comment]
Any progress on getting out a release? Is master considered stable enough for a distribution to pick up?

**Comment by Minoru** [comment]
>Any progress on getting out a release?

Nothing yet.

>Is master considered stable enough for a distribution to pick up?

Well, nothing critical has been reported against it, so I guess it is. 
I run recent(-ish) build and haven't noticed any show-stoppers myself.

If you're maintaining Newsbeuter package in some distro, check out the 
work done by Nikos Tsipinakis in Debian—he backported some important 
patches onto 2.9.

**Comment by Minoru** [comment]
> > Is master considered stable enough for a distribution to pick up?
> Well, nothing critical has been reported against it, so I guess it is.

I have to take that back. [2.10 milestone[(https://github.com/akrennmair/newsbeuter/milestone/2) shows that we have two more bugs to fix before the release.

**Comment by Minoru** [comment]
Since my last comment, two things changed:

* I got a semblance of peace with my dayjob. Now I know how little energy and time I have, and can plan around that properly. I am more ready for upheavals now :)

* Due to #591, I am becoming less perfectionist about the solutions I should employ. Specifically, I am at the brink of doing GitHub releases and posting docs somewhere other than newsbeuter.org. I am not actively working on this yet, but at least I no longer consider this a non-solution.

**Comment by klartext** [comment]
If you don't want to call a new release 2.10, because of the milestones,
just call it 2.9.a 2.9.b or something similar.
It's just a oneliner for the tagging and  a push, and you make many people happy.

Regarding the security-issue: just disable bookmarking, until there is a valid solution,
which might be called 2.9.c or something like that.

**Comment by klartext** [comment]
@Minoru: I have two different newsbeuters running here.
The one is 2.10-d1c1, the other is 2.10-e0f9.
How can that be?

**Comment by Minoru** [comment]
Guess they're using different caches.

Better open a separate issue and let's discuss it there.

**Comment by klartext** [comment]
I'm confused now.
Did you read my answer from yesterday?

**Comment by Minoru** [comment]
Yes, but I don't have an answer, so I didn't reply.

If you're confused with "2.10" in version strings, don't be-this is what Git version calls itself.
